src/files/rename-prompt/RenamePrompt.stories.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import { action } from '@storybook/addon-actions'
+import RenamePrompt from './RenamePrompt'
+
+storiesOf('Files', module)
+  .add('Rename Prompt', () => (
+    <div className='ma2'>
+      <RenamePrompt filename='my-agenda.markdown' cancel={action('Cancel')} action={action('Rename')} />
```

Comment:
`action` is a generic term; it doesn't give the reader much of an idea what it is used for. How about `onSubmit` and `onCancel` so you can see from the calling code what's expected.